test/base/TextBuilderTest.ooc

```diff
@@ -67,6 +74,9 @@ TextBuilderTest: class extends Fixture {
 			s := t take() toString()
 			expect(s == t)
 			s free()
+			sb free()
+			sb2 free()
+			sb3 free()
```

Comment:
Something like `(s, sb, sb2, sb3) free()` would be so nice, but it doesn't work.